keps/sig-apps/2599-minreadyseconds-for-statefulsets/README.md

```diff
@@ -403,16 +403,28 @@ This section must be completed when targeting beta to a release.
 Try to be as paranoid as possible - e.g., what if some components will restart
 mid-rollout?
 -->
+It shouldn't impact already running workloads. This is an opt-in feature since 
+users need to explicitly set the minReadySeconds parameter in the StatefulSet spec i.e `.spec.minReadySeconds` field.
+If the feature is disabled the field is preserved. If it was already set in the persisted StatefulSet object, otherwise it is silently dropped.
 
 ###### What specific metrics should inform a rollback?
 
 <!--
 What signals should users be paying attention to when the feature is young
 that might indicate a serious problem?
 -->
+`minReadySeconds` in StatefulSet doesn't get respected and all the `Ready` pods would be shown as `Available`. 
```

Comment:
This is not a metric; how does one measure this?

Reply:
By looking at the logs. I tried explaining it [below](https://github.com/kubernetes/enhancements/pull/2824/files#diff-08898e79412d6aad472d55e637efe7f433a78688ac74d94b6d1283b357a1a0a3R417) or should we always use a metric? I thought looking at the logs is a signal that cluster-admin or developer can use to check if the feature is working or not.